src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -89,16 +91,12 @@ internal sealed class TupleTypeSymbol : TypeSymbol, ITupleTypeSymbol
             return underlyingType;
         }
 
-        // TODO: VS does it make sense to this this two-stage?
-        // get underlying type (above),
-        // then if all ok
-        // get underlying fields here.
-        // NOTE: underlying fields do not need the underlying type and in case of an erro still can be created.
-        private ImmutableArray<TupleFieldSymbol> GetTupleFields(
-            ImmutableArray<string> elementNames, 
-            NamedTypeSymbol _underlyingType)
+        internal NamedTypeSymbol UnderlyingTupleType
```

Comment:
I do not expect, yet, that tuple need a lot of special casing to add a helper. Can add later.